Robot2019/src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -49,8 +50,12 @@ public void setLights(LightState newState) {
       color = "Orange";
       break;
       case HATCH:
-      lights.set(Relay.Value.kReverse);
-      color = "Yellow";
+      lights.set(Relay.Value.kOn);
```

Comment:
HATCH should remain kReverse. Only CLIMBER should be kOn.

Reply:
I thought blue had to be both on?